sympy/vector/coordsysrect.py

```diff
@@ -196,21 +198,32 @@ def _connect_to_standard_cartesian(self, curv_coord_type):
         :param curv_coord_type: str, tuple
 
         """
+
         if isinstance(curv_coord_type, string_types):
             self._set_transformation_equations_mapping(curv_coord_type)
             self._set_lame_coefficient_mapping(curv_coord_type)
+            if inverse:
+                self._set_inv_trans_equations(curv_coord_type)
 
         elif isinstance(curv_coord_type, (tuple, list, Tuple)) and len(curv_coord_type) == 3:
-            self._transformation_eqs = curv_coord_type
-            self._h1, self._h2, self._h3 = self._calculate_lame_coefficients(curv_coord_type)
+            if any(i.atoms(BaseScalar) for i in curv_coord_type):
```

Comment:
Because without that it could be possible to set transformation equations which doesn't depend on base scalars. 
For example, previously we could set transformation equations 
```python
a = CoordSys3D('a')
a._connect_to_standard_cartesian((x, y, z))
```
It shouldn't be possible. If someone wants to use symbols they should do
```python
a._connect_to_standard_cartesian(((x, y, z), (x, y, z)))
```